client/src/components/Chat/Conversation.tsx

```diff
@@ -1,25 +1,29 @@
 import React, { useCallback, useEffect, useRef } from 'react';
-import { ChatMessage, ChatMessageAuthor } from '../../types/general';
+import {
+  ChatMessage,
+  ChatMessageAuthor,
+  ChatMessageServer,
+} from '../../types/general';
 import Message from './ConversationMessage';
 
 type Props = {
   conversation: ChatMessage[];
   threadId: string;
-  queryId: string;
   repoRef: string;
   repoName: string;
   isLoading?: boolean;
   isHistory?: boolean;
+  onMessageEdit: (parentQueryId: string, i: number) => void;
```

Comment:
i believe some of this logic has changed: https://github.com/BloopAI/bloop/pull/780/commits/047ccbf87ed900c9aec55e4a87a4cc94073ab54e